api/application_submissions/index.js

```diff
@@ -0,0 +1,61 @@
+const express = require('express');
+
+const router = express.Router();
+
+const ApplicationSubmission = require('../../models/applicationSubmissionsModel');
+
+router.get('/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+    const submission = await ApplicationSubmission.findById(id);
+    if (submission) {
+      res.status(200).json({ submission, error: null });
+    } else {
+      res.status(400).json({ message: 'Submission not found in the database' });
+    }
+  } catch (error) {
+    res.status(500).json({ error, message: 'Unable to make request to server' });
+  }
+});
+
+router.post('/', async (req, res) => {
+  const postSubmission = {
+    ...req.body,
+    decision: 'PENDING'
+  };
+  try {
+    const [newSubmission] = await ApplicationSubmission.insert(postSubmission);
+    if (newSubmission) {
+      res.status(201).json({ newSubmission, message: 'Submission added to database' });
+    }
```

Comment:
If there's an issue, Knex should throw an error that'll be caught by the `catch` block so we can delete this conditional and just return a response.